src/plugins/data_views/server/fetcher/lib/field_capabilities/field_caps_response.ts

```diff
@@ -156,6 +157,12 @@ export function readFieldCapsResponse(
         timeSeriesMetric: timeSeriesMetricType,
         timeSeriesDimension: capsByType[types[0]].time_series_dimension,
       };
+      if (
+        capsByNameThenType['ecs.version'] &&
+        EcsFlat[fieldName as keyof typeof EcsFlat]?.description
+      ) {
+        field.ecsDescription = EcsFlat[fieldName as keyof typeof EcsFlat].description;
+      }
```

Comment:
yes, because adding it every time would make it too expensive.  maybe it could be even more granular, something like 
`includeEcsMeta=all` -> return all ECS Meta data?
`includeEcsMeta=description,short,...` -> return selected ECS Meta data?

Then it would be flexible, e.g. for the showing the description in the fields popover just single field + description could be requested. By default, when a data view requests all the fields, just a single small field could be requested, sufficient to add priority, if set

Reply:
wdyt @mattkime ?

Reply:
I will create an issue for that

Reply:
Do we expect that ECS data will be loaded for a small number of fields at a time? I think you said so much but I thought I'd verify. 

Will the ECS data be loaded for fields individually or for a small number at a time?

Does field_caps data inform how ECS data is returned?

Reply:
> Do we expect that ECS data will be loaded for a small number of fields at a time? I think you said so much but I thought I'd verify.

it depends on the use case, when showing the description of a field, yes, just a single field. But for prioritization, telemetry, this number of fields can be big (that's why the API needs to be flexible, to just serve the ECS data that's needed)

> Will the ECS data be loaded for fields individually or for a small number at a time?
depends on the use case

> Does field_caps data inform how ECS data is returned?
No, ECS data is being added / enriched on top of the field_caps result